manifests/params.pp

```diff
@@ -70,21 +78,25 @@
         # needed for mysqlproxy
         'glib2-devel',
         # needed to build zookeeper
-        #'java-1.7.0-openjdk',
+        'java-1.8.0-openjdk',
         # needed to build git
         'gettext',
         'libcurl-devel',
         'perl-ExtUtils-MakeMaker',
       ]
-      case $::operatingsystem {
-        # fedora 21 moves to openjdk 1.8.0; el6 -> f20 have 1.7.0 available
-        'Fedora': {
-          $dependency_packages = concat($base_packages, 'java-1.8.0-openjdk')
-        }
-        default: {
-          $dependency_packages = concat($base_packages, 'java-1.7.0-openjdk')
-        }
+
+      $convenience_packages = [
+        'screen',
+        'tree',
+        'vim-enhanced',
```

Comment:
Similarly with all the OSes. Definitely in favor of emacs-nox and possibly tmux.